scripts/utils.ts

```diff
@@ -350,7 +350,7 @@ export async function checkInstanceHealth(instanceId: string, region: string = "
         InstanceIds: [instanceId],
         DocumentName: "AWS-RunShellScript",
         Parameters: {
-          commands: ["screen -ls"]
+          commands: ["cd /home/ubuntu && npm run healthcheck"]
```

Comment:
npm error enoent Could not read package.json: Error: ENOENT: no such file or directory, open '/home/ubuntu/package.json'

Looks like it's not here either. Reference our ci.yaml to find the right spot